2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -151,8 +150,12 @@ function mongo_primary_member_addr() {
 
 # mongo_remove removes the current MongoDB from the cluster
 function mongo_remove() {
-  local primary_addr
-  primary_addr="$(mongo_primary_member_addr)"
+  # Declare and assign variable in the single line to ignore possible non-zero
+  # return code from mongo_primary_member_addr()
+  local primary_addr="$(mongo_primary_member_addr)"
```

Comment:
This is a subtle hack...
If we are to ignore an error, please let's at least ignore it explicitly.